esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -59,6 +59,7 @@
 #endif
 
 #define ESP_WORKUNIT_DIR "workunits/"
+const char* zipFolder = "tempzipfiles" PATHSEPSTR;
```

Comment:
trivial: should be static really, and preferrable for const like this to be a constexpr, so:
```
static constexpr const char* zipFolder = ..
```